tests/test_type_inference/test_classdef.py

```diff
@@ -31,6 +31,7 @@ def test_classdef_attribute_assign():
 
 def test_classdef_method_call():
     """Test whether type of the method call are properly being set"""
```

Comment:
This test was previously passing, but for the wrong reasons. I'll investigate this in a separate pull request.